jsonrpc/jsonrpc.go

```diff
@@ -40,6 +40,7 @@ type JSONRPC struct {
 }
 
 type dispatcher interface {
+	RemoveFilterByWs(conn wsConn) bool
```

Comment:
Why do you need to have feedback data from this method?

Reply:
fixed dfa9cf9b1ee51761e111616bca079e4d33220df0